pycue/opencue/cloud/gce_api.py

```diff
@@ -0,0 +1,145 @@
+#  Copyright (c) 2018 Sony Pictures Imageworks Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from .api import CloudInstanceGroup
+
+from googleapiclient import discovery
+from oauth2client.client import GoogleCredentials
```

Comment:
Go ahead and add these libraries to the top-level `requirements.txt` so we get tests running in the CI platform. It's not ideal to install Google-specific libraries if folks aren't using the Google versions of these plugins, but we don't have a good way to supplement optional libraries at the moment -- we can revisit that later.

Reply:
I had to added both `google-python-client` and `oauth2client` and seems like the new test  I setup ran successfully!